ci/cvmfs/snapshotter_container.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
+
+#
+# This script builds the CernVM-FS snapshotter container.
+#
+
+set -e
+
+SCRIPT_LOCATION=$(cd "$(dirname "$0")"; pwd)
+. ${SCRIPT_LOCATION}/../common.sh
+
+if [ $# -lt 2 ]; then
+  echo "Usage: $0 <CernVM-FS source directory> <build result location> [<nightly build number>]"
+  echo "This script builds CernVM-FS snapshotter container"
+  exit 1
+fi
+
+CVMFS_SOURCE_LOCATION="$1"
+CVMFS_RESULT_LOCATION="$2"
+CVMFS_NIGHTLY_BUILD_NUMBER="${3-0}"
+
+# retrieve the upstream version string from CVMFS
+cvmfs_version="$(get_cvmfs_version_from_cmake $CVMFS_SOURCE_LOCATION)"
+echo "detected upstream version: $cvmfs_version"
+
+git_hash="$(get_cvmfs_git_revision $CVMFS_SOURCE_LOCATION)"
+
+# generate the release tag for either a nightly build or a release
+CVMFS_TAG=
+if [ $CVMFS_NIGHTLY_BUILD_NUMBER -gt 0 ]; then
+  build_tag="git-${git_hash}"
+  nightly_tag="0.${CVMFS_NIGHTLY_BUILD_NUMBER}.${git_hash}git"
+
+  echo "creating nightly build '$nightly_tag'"
+  CVMFS_TAG="${cvmfs_version}-$nightly_tag"
+else
+  echo "creating release: $cvmfs_version"
+  CVMFS_TAG="${cvmfs_version}-1"
+fi
+
+for d in build rootfs; do
```

Comment:
I think this block here is necessary for creating the folders (`/build`, for example) inside the build location in which the build product is going to be available. Do you think this is a good approach @jblomer ?

Reply:
The build directory should exist and be empty when the script is started.  I think instead of this loop, the code should check if the directory exists and is empty.